src/migrations/20190829150252-create-users.js

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
why did you delete the `src/migrations/20190815004520-create-users.js` and `src/middlewares/mail.js` files?

@ola357 @conquext @OlawaleJoseph @dewaleolaoye and @Dolapo47 is there a reason why you let it slide?